app/controllers/comments_controller.rb

```diff
@@ -14,7 +14,7 @@ class CommentsController < ApplicationController
   before_filter :check_permission_to_edit, :only => [:edit, :update ]
   before_filter :check_permission_to_delete, :only => [:delete_comment, :destroy]
   before_filter :check_anonymous_comment_preference, :only => [:new, :create, :add_comment_reply]
-  before_filter :check_permission_to_review, :only => [:unreviewed]
+  before_filter :check_permission_to_review, :only => [:unreviewed, :show]
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer `before_action` over `before_filter`.